README.md

```diff
@@ -139,7 +139,6 @@ Some [Content Management System](#content-management-systems-cms) solutions also
 **[`^        back to top        ^`](#)**
 
 - [Alltube](http://www.alltubedownload.net) - Web interface for [youtube-dl](https://github.com/rg3/youtube-dl), a program to download videos and audio from [more than 100 websites](https://rg3.github.io/youtube-dl/supportedsites.html). ([Demo](http://www.alltubedownload.net), [Source Code](https://github.com/Rudloff/alltube)) `GPL-3.0` `PHP`
-- [AmIUnique](https://amiunique.org/) - Learn how identifiable you are on the Internet (browser fingerprinting tool). ([Source Code](https://github.com/DIVERSIFY-project/amiunique)) `MIT` `Java`
```

Comment:
Please don't remove AmIUnique, it is still maintained, see https://github.com/DIVERSIFY-project/amiunique/issues/51 and other issues in the repository

Reply:
Agreed!

Reply:
The url returns a 502, so at the very least there needs to be a new link, correct?

Reply:
Let's wait for a replay to https://github.com/DIVERSIFY-project/amiunique/issues/52

Reply:
https://amiunique.org is back
thanks for your patience

Reply:
Ok, added amiunique.org back into the pull request. Only site being removed is niltalk. Thanks